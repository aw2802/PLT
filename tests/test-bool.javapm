public class TestBool {

	public void main() {
		if(true and true) {
			print("true and true");
		}
		else {
			print("Nope");
		}
	}
}
