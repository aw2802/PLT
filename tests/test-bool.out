Inside truebool 
1
Inside falsebool
0
true
false
